docker-compose.yml

```diff
@@ -12,13 +12,14 @@ services:
       - 3000:3000
     volumes:
       - ./:/opt/app
-      - rails_tmp:/opt/app/tmp
     depends_on:
       - db
     environment:
       - RECAPTCHA_SITE_KEY=aaaaa
       - RAILS_ADMIN_USERNAME=rails
       - RAILS_ADMIN_PASSWORD=rails
+      - CLIENT_ID=test_0xKvM6N9
```

Comment:
These are values for a test application so there is no harm them being in source control